docs/articles_en/openvino_workflow/model_preparation/Convert_Model_From_TensorFlow.md

```diff
@@ -196,6 +196,31 @@ Converting TensorFlow Models from Memory Using Python API
 
 Model conversion API supports passing TensorFlow/TensorFlow2 models directly from memory.
 
+* ``Trackable``. The object returned by ``hub.load()`` can be converted to ``ov.Model`` with ``convert_model()``.
+
+  .. code-block:: py
+     :force:
+
+     import tensorflow_hub as hub
+     import openvino as ov
+    
+     model = hub.load("https://tfhub.dev/svampeatlas/vision/embedder/fungi_V2/1?tf-hub-format=compressed")
```

Comment:
please change a model to one with normal license. Also, please remove `?tf-hub-format=compressed` suffix